rest-api/test/unit_test/dao_test/dv_order_dao_test.py

```diff
@@ -55,3 +59,31 @@ def test_insert_biobank_order(self):
     self.assertEquals(put_response['biobankOrderId'], '12345')
     self.assertEquals(post_response['meta']['versionId'].strip('W/'), '"1"')
     self.assertEquals(put_response['meta']['versionId'].strip('W/'), '"2"')
+
+  def test_enumerate_shipping_status(self):
+    fhir_resource = SimpleFhirR4Reader(self.post_request)
+    status = self.dao._enumerate_order_shipping_status(fhir_resource.status)
+    self.assertEquals(status, OrderShipmentStatus.SHIPPED)
+
+  def test_enumerate_tracking_status(self):
+    fhir_resource = SimpleFhirR4Reader(self.post_delivery)
+    status = self.dao._enumerate_order_tracking_status(fhir_resource.extension.get(url=VIBRENT_FHIR_URL + 'tracking-status').valueString)
+    self.assertEquals(status, OrderShipmentTrackingStatus.ENROUTE)
+
+  @mock.patch('dao.dv_order_dao.MayoLinkApi')
+  def test_service_unavailable(self, mocked_api):
+    def raises(*args):
+      print args
```

Comment:
we should remove the extra `print`

Reply:
I agree, but I could not get #pylint disable=unused-argument to work. I put it on every line ! Hence, the "still trying" commit comments above. Doing this was the only thing I could find to shut it up 🤷🏻‍♂️

Reply:
Try one with a `:` after `#pylint`, if it still complains I'll approve as-is

```suggestion
      #pylint: disable=unused-argument
```

Reply:
🙄done, thanks.